tests/integration_tests/functional/test_drives.py

```diff
@@ -96,7 +96,10 @@ def test_rescan_dev(test_microvm_with_ssh, network_config):
     # https://github.com/firecracker-microvm/firecracker-containerd/issues/141
     loopback_device = f'/dev/loop{os.getpid()}'
     losetup = ['losetup', loopback_device, fs2.path]
-    run(losetup, check=True)
+    try:
+        run(losetup, check=True)
+    except CalledProcessError:
+        pytest.skip("the kernel doesn't have enough loopback devices")
```

Comment:
I think we should add to this message a hint when the not so common case for the failure happens, otherwise, it will be misleading to know just that the skipping is occurring only because we can not create another loopback device.

Reply:
Will capture stdin/stdout and include that on the skip message.